shared/hoverAlerts.ts

```diff
@@ -0,0 +1,84 @@
+import * as sourcegraph from 'sourcegraph'
+
+export const HoverAlerts = {
+    LSIF: <sourcegraph.Badged<sourcegraph.HoverAlert>[]>[
+        {
+            summary: {
+                kind: sourcegraph.MarkupKind.Markdown,
+                value:
+                    'Semantic result. [Learn more.](https://docs.sourcegraph.com/user/code_intelligence/lsif)',
+            },
+            badge: {
+                kind: 'info',
+                hoverMessage:
+                    "This hover text comes from a pre-computed semantic index of this project's source. Click to learn how to add this capability to all of your projects!",
```

Comment:
'hover text' may not be clear or accurate. Alternative ideas: 
- "This code intelligence data"
- "This hover data"
- "This reference data"

We have some style guide info on wording of links, though this is a bit difference since it's in a hover: https://about.sourcegraph.com/handbook/communication/style_guide#links

Reply:
maybe just "text"

Reply:
Also, I love that you ended with an exclamation mark. This is *exactly* the kind of enthusiasm I'm trying to get you to express in the release blog content. =)

Reply:
@LaureenH - the reason I made alternate suggestions is that "text" feels inaccurate to me. This is because it'ts not just copy. It's information about your code including the function definition, parameters and types, as well as any available documentation in the code. Here's what it looks like: 
![image](https://user-images.githubusercontent.com/1559622/87585095-5d896900-c693-11ea-9e1e-c0a30948fdc1.png)

Reply:
other ideas:
- 'content'/'hover content'
- 'symbol information'

I favour 'text' because I like concision, or 'symbol information' for maximum specificity, thoughts?

Reply:
I went with 'hover data', symbol information is too specific, and the hovers in general could contain content other than text.